CONTRIBUTING.md

```diff
@@ -10,4 +10,26 @@ be prefixed with a package name (`{PACKAGE_NAME}-.+\.json`).
 
 Currently, the following placeholders are available:
 
-* `{PACKAGE_NAME}` - name of the package
\ No newline at end of file
+* `{PACKAGE_NAME}` - name of the package
+
+## Folder Item schema
+
+### Defining property format
+
+The JSON Schema defines the basic structure of a JSON document (e.g. package manifests, ingest pipelines, etc.).
+In some cases this might be insufficient as there are properties that require strict validation (not just type
+consistency), e.g. format validation:
+
+```yaml
+src:
+  description: Relative path to the screenshot's image file.
+  type: string
+  format: relative-path
+  examples:
+  - "/img/apache_httpd_server_status.png"
+```
+
+Currently, the following custom formats are available:
+
+* `relative-path`: Relative path to the package root directory. The format checker verifies if the path is correct and
```

Comment:
Is it relative to the package root directory or is it relative to where the folder item spec file is defined?

Reply:
It is relative to the package root (as noted).